### PR TITLE
Use currently unused local $reflection var

### PR DESCRIPTION
**Pull request by nicolas-grekas**

### CODE REVIEW

**Comment by Ocramius** [comment]
Marking as `bug`, because this was supposed to be a performance optimisation that isn't being taken advantage of (benchmark test suite will say something about this).

**Comment by nicolas-grekas** [comment]
> Wow, how did I miss this one? :O

Not strict enough? :P

File: src/ProxyManager/ProxyGenerator/AccessInterceptorValueHolder/MethodGenerator/StaticProxyConstructor.php [hunk 1]
```diff
@@ -61,7 +61,7 @@ public function __construct(
         $this->setBody(
             'static $reflection;' . "\n\n"
             . '$reflection = $reflection ?: $reflection = new \ReflectionClass(__CLASS__);' . "\n"
```
Comment (Majkl578) [inline src/ProxyManager/ProxyGenerator/AccessInterceptorValueHolder/MethodGenerator/StaticProxyConstructor.php]: This should be `??`, right?

Comment (Ocramius) [reply src/ProxyManager/ProxyGenerator/AccessInterceptorValueHolder/MethodGenerator/StaticProxyConstructor.php]: Most likely, yes

**Comment by Ocramius** [comment]
> Not strict enough? :P

Possibly. I need to give you a badge for finding so many bugs in the most annoyingly-tested lib I have built  so far :-)

**Comment by Ocramius** [comment]
I'll likely update the package to run with the latest PHPStan release over the weekend - the build is a proper wreckage.

**Comment by Ocramius** [comment]
Alright, won't get to fix travis for 2.x with my current time, so merging up ti `3.x` (`master`) and backporting manually :+1:

**Comment by Ocramius** [comment]
Merged into master via 4b92810f27965ff853ca29e62f976e0ca7f977d2

**Comment by Ocramius** [comment]
Thanks @nicolas-grekas!

**Pull request merged**